packages/framework/src/Model/Product/Parameter/ParameterValue.php

```diff
@@ -34,6 +35,12 @@ class ParameterValue
      */
     protected $text;
 
+    /**
+     * @var string|null
+     * @ORM\Column(type="decimal", precision=20, scale=6, nullable=true)
+     */
+    protected $numericValue;
```

Comment:
based on the acceptance criteria, this value should now be used [here](https://github.com/shopsys/shopsys/blob/15.0/project-base/app/src/Model/Product/Elasticsearch/ProductExportRepository.php#L457), right?

I do not want to speak about the fact the linked code is in `project-base` :melting_face: